arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -182,8 +181,6 @@ def validateXbrlFinally(val, *args, **kwargs):
     _xhtmlNs = "{{{}}}".format(xhtml)
     _xhtmlNsLen = len(_xhtmlNs)
     modelXbrl = val.modelXbrl
-    # reset environment formula run IDs
-    modelXbrl.modelManager.formulaOptions.runIDs = val.priorFormulaOptionsRunIDs
```

Comment:
@hefischer I can move this to `validateFormulaFinished`, but there wasn't an obvious or documented reason to reset them. Should this be moved to `validateFormulaFinished`?